x-pack/platform/plugins/shared/content_connectors/server/services/index.ts

```diff
@@ -160,7 +159,7 @@ export class AgentlessConnectorsInfraService {
     return policiesMetadata;
   };
 
-  public deployConnector = async (connector: ConnectorMetadata): Promise<PackagePolicy> => {
+  public deployConnector = async (connector: ConnectorMetadata) => {
```

Comment:
nitpick: would be good to see a return type here Promise<AgentPolicyServiceInterface['createWithPackagePolicies']>